custom.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+
+	"github.com/grupozap/aegir/internal/pkg/utils"
+	livr "github.com/k33nice/go-livr"
+)
+
+var (
+	defaultRegex = regexp.MustCompile(".*")
+)
+
+func neq(args ...interface{}) livr.Validation {
+	not_allowed := utils.FirstArg(args...)
+
+	return func(value interface{}, builders ...interface{}) (interface{}, interface{}) {
+		if value == nil || value == "" {
+			return nil, nil
+		}
```

Comment:
if `not_allowed` is `nil` you could return right after, couldn't you?

```suggestion
	not_allowed := utils.FirstArg(args...)

	return func(value interface{}, builders ...interface{}) (interface{}, interface{}) {
		if value == nil || value == "" {
			return nil, nil
		}
		
		if not_allowed == nil {
			return value, nil
		}
```

Reply:
Yes, I think I coud do it. I'll test it and try!